src/views/ChooseAddress.vue

```diff
@@ -60,10 +62,15 @@ export default class ChooseAddress extends Vue {
             return;
         }
 
-        const accountInfo = walletInfo.accounts.get(address);
+        let accountInfo: AccountInfo | ContractInfo | undefined = walletInfo.accounts.get(address);
         if (!accountInfo) {
-            console.error('Selected address not found:', address);
-            return;
+            // Search contracts
+            accountInfo = walletInfo.findContractByAddress(Nimiq.Address.fromUserFriendlyAddress(address));
```

Comment:
I see code duplication here. Could it make sense to have `walletInfo.accountsAndContracts.get(address)`, maybe with a better name?

Reply:
I fixed it differently: https://github.com/nimiq/accounts/pull/186/commits/dfa8f4e7e520779f8f058a13c197d3a52da44a58#diff-a17003124ee17fdee72e5cac1d4a52e9L65